jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
```

Comment:
Structure is copied from the logger factory, however mq-clients have it as a separate dir. What are benefits to having them separated apart from the clear distinction between production and dev code?

Reply:
Mixing test suites and source code sounds like source could end up being a mess to me. Sure, you could do a separate directory for tests within source director (jest for example assumes that tests are placed in `__tests__` directory). If that's the case however, it could be decided and what convention should be followed for such tests.